<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>充电口识别</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>charger.ico</normaloff>charger.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2" colspan="2">
       <widget class="QTextEdit" name="textEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>4</horstretch>
          <verstretch>60</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
border-style: solid;
border-color: #55aaff;
border-radius: 2px;
background-color: rgb(245, 245, 255);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="6">
       <widget class="QVTKOpenGLNativeWidget" name="openGLWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>30</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 3px;
border-color: #55aaff;
border-radius: 5px;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSlider" name="ySlider">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Y</string>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pushButton_next">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>false</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>跳过</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pushButton_manual">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>false</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>手动</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="xSlider">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>X</string>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="pushButton_auto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>false</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>自动</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSlider" name="rxSlider">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>rx</string>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="pushButton_action">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>false</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>充电</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSlider" name="rySlider">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>ry</string>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSlider" name="zSlider">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Z</string>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSlider" name="rzSlider">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>rz</string>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>显示设置</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionshow_coordinate"/>
    <addaction name="separator"/>
    <addaction name="actionshow_camera"/>
    <addaction name="separator"/>
    <addaction name="actionshow_ur5"/>
    <addaction name="separator"/>
    <addaction name="actionclear_text"/>
    <addaction name="separator"/>
    <addaction name="actionswitch_mode"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>关于软件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_aboutapp"/>
    <addaction name="separator"/>
    <addaction name="action_aboutme"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>外设连接</string>
    </property>
    <addaction name="separator"/>
    <addaction name="connect_camera3d"/>
    <addaction name="separator"/>
    <addaction name="connect_ur5"/>
    <addaction name="separator"/>
    <addaction name="reset_ur5"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>手眼标定</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actioncapture"/>
    <addaction name="separator"/>
    <addaction name="actionmove_arm"/>
    <addaction name="separator"/>
    <addaction name="actionoffset"/>
    <addaction name="separator"/>
    <addaction name="actionsave_offset"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionshow_coordinate">
   <property name="text">
    <string>显示坐标轴</string>
   </property>
  </action>
  <action name="actionshow_camera">
   <property name="text">
    <string>显示3d相机</string>
   </property>
  </action>
  <action name="actionshow_ur5">
   <property name="text">
    <string>显示机械臂</string>
   </property>
  </action>
  <action name="action_aboutapp">
   <property name="text">
    <string>关于软件</string>
   </property>
  </action>
  <action name="action_aboutme">
   <property name="text">
    <string>关于作者</string>
   </property>
  </action>
  <action name="connect_camera3d">
   <property name="text">
    <string>连接相机</string>
   </property>
  </action>
  <action name="connect_ur5">
   <property name="text">
    <string>连接机械臂</string>
   </property>
  </action>
  <action name="reset_ur5">
   <property name="text">
    <string>复位机械臂</string>
   </property>
  </action>
  <action name="actioncapture">
   <property name="text">
    <string>执行采集</string>
   </property>
  </action>
  <action name="actionmove_arm">
   <property name="text">
    <string>移动机械臂</string>
   </property>
  </action>
  <action name="actionclear_text">
   <property name="text">
    <string>清空文本框</string>
   </property>
  </action>
  <action name="actionoffset">
   <property name="text">
    <string>打开手眼补偿</string>
   </property>
  </action>
  <action name="actionsave_offset">
   <property name="text">
    <string>保存手眼补偿</string>
   </property>
  </action>
  <action name="actionswitch_mode">
   <property name="text">
    <string>模式切换</string>
   </property>
   <property name="toolTip">
    <string>切换钻孔或充电模式</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>qvtkopenglnativewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
